OpenRA.Game/Widgets/WorldInteractionControllerWidget.cs

```diff
@@ -224,6 +255,14 @@ public override bool HandleKeyPress(KeyInput e)
 			return false;
 		}
 
+		static IEnumerable<Actor> DeadzoneSelectBox(World world, int2 a, int2 b, Func<Actor, bool> cond)
```

Comment:
It would be nice if the name was consistent with `SelectActorsInBox`.  Maybe `SelectActorsInBoxWithDeadZone`?